pkger/parser_models.go

```diff
@@ -977,6 +977,7 @@ const (
 )
 
 type color struct {
+	ID   string `json:"id,omitempty" yaml:"id,omitempty"`
```

Comment:
just checking that there shouldn't be a situation where the ID would empty / 0, right?

Reply:
swagger shouldn't need to be updated as this is only preserving the already present color id. it was previously getting dropped and it appears the ui relies on it (no console errors though) to be set to show that color (it's been `base` in every dashboard i looked at).

Reply:
if it is empty, it would be when `apply`ing an old `export` where the id was not set with the color information. the color would still appear broken because i didn't default empty id's to `"base"`, but if the user fixes the color and re-exports, the color would be preserved by this change.